tools/model_tools/src/openvino/model_zoo/open_model_zoo.py

```diff
@@ -0,0 +1,293 @@
+
+    def vocab(self):
+        model = self._load_model(self.name)
+        vocab_dir = self.download_dir / self.subdirectory
+        if model:
+            for file in model.files:
+                if 'vocab' in file.name.name:
+                    vocab_path = vocab_dir / file.name
+                    break
+        else:
+            if (vocab_dir / 'vocab.txt').exists():
+                vocab_path = vocab_dir / 'vocab.txt'
+            else:
+                vocab_path = vocab_dir / 'vocab.json'
+
+        if vocab_path.exists():
+            return self.load_vocab_file(vocab_path)
+        else:
+            return None
+
+    def inputs(self):
+        if self.model is None:
+            try:
+                self.model = self.ie.read_model(self.model_path)
+            except AttributeError:
+                raise TypeError('ie is not specified or is of the wrong type.'
+                                'Please check ie is of openvino.runtime.Core type.')
+
+        return self.model.inputs
+
+    def outputs(self):
+        if self.model is None:
+            try:
+                self.model = self.ie.read_model(self.model_path)
+            except AttributeError:
+                raise TypeError('ie is not specified or is of the wrong type.'
+                                'Please check ie is of openvino.runtime.Core type.')
+
+        return self.model.outputs
+
+    def preferable_input_shape(self, name):
+        input_info = self.model_config().get('input_info', [])
+        for input in input_info:
+            if input['name'] == name:
+                return input['shape']
+
+        return None
+
+    def layout(self, name):
+        input_info = self.model_config().get('input_info', [])
+        for input in input_info:
+            if input['name'] == name:
+                return input['layout']
+
+        return None
+
+    def input_info(self):
+        return self.model_config().get('input_info', [])
+
+    def _create_model_api_instance(
+        self, model_creator, num_streams='', num_threads=None, max_num_requests=1, configuration=None
+    ):
+        plugin_config = _common.get_user_config(self.device, num_streams, num_threads)
+        model_adapter = _common.OpenvinoAdapter(_common.create_core(), self.model_path, device=self.device,
+                            plugin_config=plugin_config, max_num_requests=max_num_requests)
+        self.model_api = model_creator(model_adapter, configuration)
+        self.model_api.load()
+
+    def model_api_inference(self, inputs, device='CPU', model_creator=None, **kwargs):
+        self.device = device
+        if model_creator is None:
+            model_creator = self.model_creator
+
+        self._create_model_api_instance(model_creator, **kwargs)
+
+        preprocessed_inputs, meta = self.model_api.preprocess(inputs)
+        raw_results = self.model_api.infer_sync(preprocessed_inputs)
+        results = self.model_api.postprocess(raw_results, meta)
```

Comment:
```suggestion
        results = self.model_api(inputs)
```
Since the Python Model API wrappers have `__call__()` operator, which performs the same actions